src/esm_runscripts/filedicts.py

```diff
@@ -122,20 +122,21 @@ def __init__(self, full_config: dict, attrs_address: dict):
         self.name = attrs_address.split(".")[-1]
         self.component = component = attrs_address.split(".")[0]
         self.all_model_filetypes = full_config["general"]["all_model_filetypes"]
+        self.path_in_computer = self.get("path_in_computer")
+        if self.path_in_computer:
+            self.path_in_computer = pathlib.Path(self.path_in_computer)
 
         self._check_file_syntax()
 
-        self.path_in_computer = pathlib.Path(self["path_in_computer"])
-
         # Complete tree names if not defined by the user
-        self["name_in_run_tree"] = self.get(
-            "name_in_run_tree", self["name_in_computer"]
-        )
-        self["name_in_exp_tree"] = self.get(
-            "name_in_exp_tree", self["name_in_computer"]
-        )
-        if self["type"] not in ["restart", "outdata"]:
-            self["name_in_work"] = self.get("name_in_work", self["name_in_computer"])
+        if self["type"] in self.input_file_types:
+            default_name = self["name_in_computer"]
```

Comment:
Not `"name_in_exp_tree"`...?

Reply:
I think the users think more of things coming from the pool to the experiment/work, not from the exp_dir to the work directory.